app/controllers/specialist_documents_controller.rb

```diff
@@ -46,4 +46,8 @@ def base_path
   def error_403(exception)
     render plain: exception.message, status: 403
   end
+
+  def content_store
+    @content_store ||= GdsApi::ContentStore.new(Plek.current.find("content-store"))
+  end
```

Comment:
i was just following what government-frontend had, i couldn't find the magic, will try without